### PR TITLE
Opportunities screen first part

### PR DESCRIPTION
**Pull request by Viacheslav83**

### CODE REVIEW

File: INTITA/Models/OpportunitiesModel.swift [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        case .course:
+            return "information_about_course".localized
+        case .task:
+            return "answer_assessments".localized
+        case .study:
+            return "courses_modules_etc".localized
+        }
+    }
+    
+    var sectionButonTask: String {
```
Comment (svitlana-korostelova) [inline INTITA/Models/OpportunitiesModel.swift:39]: rename please, if it "button" I assume

File: INTITA/Modules/OpportunitiesScreen/OpportunitiesViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,19 @@
+//
+//  OpportunitiesViewModel.swift
+//  INTITA
+//
+//  Created by Viacheslav Markov on 24.01.2021.
+//
+
+import Foundation
+
+class OpportunitiesViewModel {
```
Comment (svitlana-korostelova) [inline INTITA/Modules/OpportunitiesScreen/OpportunitiesViewModel.swift:10]: According to MVVM, it should have a link to OpportunitiesModel.

Comment (Viacheslav83) [reply INTITA/Modules/OpportunitiesScreen/OpportunitiesViewModel.swift]: Thanks for advice. During the improvement of functionality the OpportunitiesViewModel was deleted.

File: INTITA/Models/OpportunitiesModel.swift (hunk 1 above)
Comment (Viacheslav83) [reply INTITA/Models/OpportunitiesModel.swift]: It has already removed.

**Review by oleksiikolomiietssnapp — APPROVED** [review]

File: INTITA/Models/OpportunitiesType.swift [hunk 3]
```diff
@@ -0,0 +1,55 @@
+//
+//  OpportunitiesModel.swift
```
Comment (svitlana-korostelova) [inline INTITA/Models/OpportunitiesType.swift:2]: You can rename the comment too, but it's minor. And replace this file from Models into Modules/OpportunitiesScreen

**Review by svitlana-korostelova — APPROVED** [review]

File: INTITA/Models/OpportunitiesType.swift (hunk 3 above)
Comment (Viacheslav83) [reply INTITA/Models/OpportunitiesType.swift]: Sorry. My bad. I renamed it.

**Pull request merged**